cmd/root.go

```diff
@@ -75,15 +77,24 @@ the latest stable version will be downloaded from the repository.
 				// will be parsed correctly.
 				// e.g: tiup --tag mytag --rm playground --db 3 --pd 3 --kv 4
 				//   => run "playground" with parameters "--db 3 --pd 3 --kv 4"
+				// tiup --tag mytag --binpath /xxx/tikv-server tikv
 				var transparentParams []string
 				componentSpec := args[0]
 				for i, arg := range os.Args {
 					if arg == componentSpec {
-						transparentParams = os.Args[i+1:]
+						var num = 1
+						if len(os.Args) > i+1 && strings.HasPrefix(os.Args[i+1], "--binpath=") {
```

Comment:
I think we should not filter the `--binpath` in this way because of the `--binpath` has been parsed by the cobra.
The syntax of running a component is `tiup [flags] <component> [args]`.